bindings/python/crocoddyl/multibody/costs/contact-impulse.cpp

```diff
@@ -0,0 +1,93 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020-2022, University of Edinburgh
```

Comment:
Why this 2022?

Reply:
I assumed we define two years ahead copyright, is this not the correct way of doing?

Reply:
It is not. This line is about authorship: who authored, and when. You can't author something from the future. If **you** wrote this in **2020**, you should put 2020 - (your employer).

If you did not write anything in a file, you can't claim authorship on this file, and if you didn't write anything in 2022 in that file, you can't claim authorship on this file for that year.

Reply:
Thank you for the clarification.
I fixed it.